tasks/copy_to_glacier/copy_to_glacier.py

```diff
@@ -76,6 +76,7 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
 
         Environment Variables:
             ORCA_DEFAULT_BUCKET (string, required): Name of the default ORCA S3 Glacier bucket.
+                Overridden by bucket specified in config.
```

Comment:
Not sure if this is needed. maybe simplify to `Name of the default ORCA S3 Glacier bucket which can be overridden  by configuration settings.` Thoughts?

Reply:
That would actually be a longer description, and drops the reference to file name 'config'.

Reply:
I guess I didn't make the files config connection reading it. I am still not convinced that we need to state how or where it is is overridden here. That may be better served in the description above where we tell what it does and the tasks behavior. Thoughts?

Reply:
I would prefer to keep the explanation of environment variables centralized.

Reply:
Also, unsure what above description you are referring to.

Reply:
Referring to the overall definition starting on line 74. So changing `Copies the files in {event}['input']
    to the ORCA glacier bucket defined in ORCA_DEFAULT_BUCKET.` to `Copies the files in {event}['input']
    to the ORCA glacier bucket defined in ORCA_DEFAULT_BUCKET overridden by setting the orcaDefaultBucketOverride variable in the configuration at the collection or step function level.`